up42/workflow.py

```diff
@@ -603,9 +631,10 @@ def _helper_run_parallel_jobs(
 
     def test_job(
         self,
-        input_parameters: Union[Dict, str, Path] = None,
+        input_parameters: Union[Dict, str, Path],
```

Comment:
Why does this need to change? I see there is some handling of None in the previous function, maybe it would be good to add the same for estimation?

Reply:
Thanks, will revert this, I had some pylint issue and when I fixed it I forget to revert this back.